src/Avatar.js

```diff
@@ -63,6 +67,7 @@ const styles = {
 };
 
 Avatar.defaultProps = {
+  alwaysRenderAvatar: false,
```

Comment:
Cool! Looks better now. I was thinking a shorter name but no idea came to my mind ;)